src/main/java/org/broadinstitute/consent/http/service/ElasticSearchService.java

```diff
@@ -383,6 +396,15 @@ public DatasetTerm toDatasetTerm(Dataset dataset) {
     return term;
   }
 
+  protected void updateDatasetIndex(Integer datasetId, Integer userId, Instant indexDate)
```

Comment:
The method name is confusing to me, since it doesn't update the index but updates the index date
```suggestion
  protected void updateDatasetIndexDate(Integer datasetId, Integer userId, Instant indexDate)
```

The same is true for `DatasetServiceDAO.updateDatasetIndex()`.